tools/harness-simulation/harness/Web/data/deviceInputFields.xml

```diff
@@ -0,0 +1,11 @@
+<DEVICE_FIELDS>
+    <DEVICE name="OpenThread_Sim" thumbnail="OpenThread.png" description = "OpenThread Simulation" THCI="OpenThread_Sim">
+        <Hardware thumbnail="Raspberry_Pi_4B.jpg", description="OpenThread Simulation FTD"></Hardware>
+        <ITEM label="Serial Line"
+              type="text"
+              forParam="SerialPort"
+              validation="COM"
+              hint="eg: COM1">COM1
```

Comment:
The item will be displayed as `<nodeid>@<ipaddr>` on the webpage. It is necessary if you want to designate a specific device as the DUT, otherwise there is no way to differentiate OT FTD simulations. The default value `COM1` will not be revealed by users unless they try to add devices manually. `COM1` here can be seen as a placeholder.